src/windshaft/dashboard.js

```diff
@@ -56,17 +56,18 @@ WindshaftDashboard.prototype._createInstance = function(options) {
         urls: dashboardInstance.getTiles('mapnik')
       });
 
-      this._updateWidgetURLs({
-        layerId: options.layerId
-      });
-
       // update other kind of layers too
-      this.layers.each(function(layer) {
+      this.layers.each(function(layer, layerIndex) {
         if (layer.get('type') === 'torque') {
+          layer.set('meta', dashboardInstance.getLayerMeta(layerIndex));
```

Comment:
The meta information that is required to have the column type only works for Torque for now? I guess this is related to why the default time-series widget doesn't work?